src/utils/function/isNaN/isNaN.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Checks if the provided value is NaN.
+ *
+ * @param {any} value - The value to check.
+ * @returns {boolean} Returns true if the value is NaN, otherwise false.
+ *
+ * @example
+ * isNaN(NaN);    // true
+ * isNaN(123);    // false
+ */
+let isNaN = (value: any): boolean => {
```

Comment:
1. use const instead of let, we are reassigning this.
2. don't add `any` not good for type safety, `unknown` will give better type safety 

```suggestion
let isNaN = (value: unknown): boolean => {
```